src/bin/vip-config-software-update.js

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env node
+
+/**
+ * @flow
+ * @format
+ */
+
+/**
+ * External dependencies
+ */
+import chalk from 'chalk';
+import debugLib from 'debug';
+
+/**
+ * Internal dependencies
+ */
+import command from 'lib/cli/command';
+import { appQuery, appQueryFragments } from 'lib/config/software';
+import { getUpdateResult, promptForUpdate, triggerUpdate } from '../lib/config/software';
+import { ProgressTracker } from '../lib/cli/progress';
+import UserError from '../lib/user-error';
+
+const debug = debugLib( '@automattic/vip:bin:config-software' );
+
+const UPDATE_SOFTWARE_PROGRESS_STEPS = [
+	{ id: 'trigger', name: 'Trigger software update' },
+	{ id: 'process', name: 'Process software update' },
+];
+
+const cmd = command( {
+	appContext: true,
+	appQuery,
+	appQueryFragments,
+	envContext: true,
+	wildcardCommand: true,
+} );
+cmd.option( 'force', 'Auto-confirm update' );
```

Comment:
Does `--force` follow the conventions we have in other places?
I know we use `-y` in some places, in WP-CLI it's `--skip-confirm`
 cc @yolih

Reply:
Oh my:

```
vip sync --help | grep force
    -f, --force    Skip confirmation (disabled by default)

vip import sql --help | grep force
    -f, --force           Skip confirmation (disabled by default)

vip wp --help | grep yes
    -y, --yes      Run the command in production without a confirmation prompt

vip config envvar set --help | grep skip
    -s, --skip-confirmation  Skip manual confirmation of input (USE WITH CAUTION) (disabled by default)
```

Reply:
oh my indeed 😬

Reply:
I see more `--force`, so we are starting the great unification!